konnectd/ui/src/components/Loading.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
This would create extra dependency on Vue, wouldn't it? At least when looking at https://www.npmjs.com/package/react-vuex that's the feeling I am getting